TESTING.md

```diff
@@ -522,3 +522,8 @@ Repeat the above starting from different themes.
 #### Expected behaviour
 
 Changing theme should lead to full repaint of the colours according to the current theme.
+
+### Guardrails:
+
+- [ ] Check if Guardrails check passes for any code longer than 10 lines generated in Cody Chat
+- [ ] Check if Guardrails check failes for any code longer than 10 lines that matches indexed OSS
```

Comment:
`that matches indexed OSS` - what is indexed OSS?
Please let's make testing guide self-eplainatory if possible.

Reply:
For now I dropped this part as I think due to editor changes ([link](https://sourcegraph.slack.com/archives/C05AGQYD528/p1713260328388809)) this is now very hard to test.